packages/devtools_app/lib/src/screens/inspector/layout_explorer/flex/flex.dart

```diff
@@ -42,41 +42,41 @@ class _FlexLayoutExplorerWidgetState extends LayoutExplorerWidgetState<
     FlexLayoutExplorerWidget, FlexLayoutProperties> {
   final scrollController = ScrollController();
 
-  Axis get direction => properties.direction;
+  Axis get direction => properties!.direction;
 
-  ObjectGroup get objectGroup =>
-      properties.node.inspectorService as ObjectGroup;
+  ObjectGroup? get objectGroup =>
+      properties!.node.inspectorService as ObjectGroup?;
 
   Color horizontalColor(ColorScheme colorScheme) =>
-      properties.isMainAxisHorizontal
+      properties!.isMainAxisHorizontal
           ? colorScheme.mainAxisColor
           : colorScheme.crossAxisColor;
 
-  Color verticalColor(ColorScheme colorScheme) => properties.isMainAxisVertical
+  Color verticalColor(ColorScheme colorScheme) => properties!.isMainAxisVertical
       ? colorScheme.mainAxisColor
       : colorScheme.crossAxisColor;
 
   Color horizontalTextColor(ColorScheme colorScheme) =>
-      properties.isMainAxisHorizontal
+      properties!.isMainAxisHorizontal
           ? colorScheme.mainAxisTextColor
           : colorScheme.crossAxisTextColor;
 
   Color verticalTextColor(ColorScheme colorScheme) =>
-      properties.isMainAxisVertical
+      properties!.isMainAxisVertical
           ? colorScheme.mainAxisTextColor
           : colorScheme.crossAxisTextColor;
 
-  String get flexType => properties.type;
+  String get flexType => properties!.type;
 
   @override
-  RemoteDiagnosticsNode getRoot(RemoteDiagnosticsNode node) {
+  RemoteDiagnosticsNode? getRoot(RemoteDiagnosticsNode? node) {
     if (!shouldDisplay(node)) return null;
```

Comment:
add an explicit 
if node == null return null
consider making shouldDisplay take a non-nullable node.
That style is a bit more consistent with null safety than the previous style of having a few too many methods that implicitly check for null to be safe.

Reply:
done